pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/TopicsImpl.java

```diff
@@ -368,9 +368,10 @@ public void revokePermissions(String topic, String role) throws PulsarAdminExcep
     }
 
     @Override
-    public void createPartitionedTopic(String topic, int numPartitions) throws PulsarAdminException {
+    public void createPartitionedTopic(String topic, int numPartitions, Map<String, String> metadata)
+            throws PulsarAdminException {
         try {
-            createPartitionedTopicAsync(topic, numPartitions).get(this.readTimeoutMs, TimeUnit.MILLISECONDS);
+            createPartitionedTopicAsync(topic, numPartitions, metadata).get(this.readTimeoutMs, TimeUnit.MILLISECONDS);
```

Comment:
suggest to use new method to pack this piece
https://github.com/apache/pulsar/blob/9994614173205abd075fcc670396cebd71227047/pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/BaseResource.java#L290
such as
https://github.com/apache/pulsar/blob/9994614173205abd075fcc670396cebd71227047/pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/NamespacesImpl.java#L244-L246

Reply:
Ok, thanks for the suggestion.